src/plugins/workflow_designer/src/ItemViewStyle.cpp

```diff
@@ -316,8 +314,10 @@ bool ExtendedProcStyle::sceneEventFilter(QGraphicsItem *watched, QEvent *event)
         ret = updateCursor(he->pos());
     } break;
     case QEvent::GraphicsSceneMouseRelease:
-        desc->mouseReleaseEvent(dynamic_cast<QGraphicsSceneMouseEvent *>(event));
     case QEvent::GraphicsSceneHoverLeave:
+        if (event->type() == QEvent::GraphicsSceneMouseRelease) {
```

Comment:
This is a bit strange, I think, the previous version was better. Restore, please

Reply:
The original code has a reasonable warning: non-empty fall-thru switch cases are bad.
I reworked the code to avoid merging of these 2 cases completely.